app/src/main/java/com/keylesspalace/tusky/service/SendStatusService.kt

```diff
@@ -290,6 +310,9 @@ class SendStatusService : Service(), Injectable {
             notificationManager.cancel(statusId)
             notificationManager.notify(errorNotificationId++, notification)
         }
+
+        // NOTE only this removes the "Sending..." notification (added with startForeground() above)
```

Comment:
Another collateral fix. Otherwise the "Sending..." notification cannot be remove upon errors.